commands/profile.go

```diff
@@ -176,6 +179,8 @@ func init() {
 		"\nIf security is disabled, provide --auth-type='disabled'.\nIf security uses HTTP basic authentication, provide --auth-type='basic'.\n"+
 		"If security uses AWS IAM ARNs as users, provide --auth-type='aws-iam'.\nodfe-cli asks for additional information based on your choice of authentication type.")
 	_ = createProfileCmd.MarkFlagRequired(FlagProfileCreateAuthType)
+	createProfileCmd.Flags().IntP(FlagProfileMaxRetry, "m", 3, "Specifies a value of maximum retry attempts the odfe-cli retry handler can perform.\n"+
+		"You can override this value by using the ODFE_MAX_RETRY environment variable.")
```

Comment:
How about "PROFILE_MAX_RETRY" rather than ODFE_MAX_RETRY?
Two concerns

1. ODFE_MAX_RETRY seem to be generic. Is this intended?
2. Avoid ODFE in the name if possible. This will help agnostic of Organization name for future changes

Reply:
Most of the environment variables supported by an application always uses application name as prefix, example: JAVA_HOME, GOPATH, GOPROXY, AWS_MAX_ATTEMPTS, AWS_RETRY_MODE, HADOOP_CONFIG, thats the reason for ODFE as prefix.

Reply:
Makes sense